src/GitVersion.App/GitVersionExecutor.cs

```diff
@@ -56,8 +57,16 @@ public int Execute(GitVersionOptions gitVersionOptions)
 
         private int RunGitVersionTool(GitVersionOptions gitVersionOptions)
         {
+            var mutexName = gitVersionOptions.WorkingDirectory.Replace("\\", "");
```

Comment:
If there is no need to make them readable of some sort you could also just hash the path into a guid